tools/checklist_package_imports.dart

```diff
@@ -0,0 +1,765 @@
+// Import the required Dart I/O package
+import 'dart:io';
+
+String createCaseCondition(String key, String value) {
+  return 'case "$key":\n    $value';
+}
+
+void insertCaseCondition(List<String> lines, String caseCondition) {
+  final defaultIndex = lines.indexWhere((line) => line.contains('default:'));
+  if (defaultIndex != -1 &&
+      !lines.any((line) => line.contains(caseCondition.split('\n')[0]))) {
+    lines.insert(defaultIndex, caseCondition);
+  }
+}
+
+// Define the main function
+void main() {
+  // Get the current directory path
+  var appDir = "/home/admin1/Downloads/Flutter/HCM_final/";
```

Comment:
This shouldn't hard-coded since it will different each user. Please change it.

Reply:
> This shouldn't hard-coded since it will different each user. Please change it.

Yes, done